modules/nf-core/bedtools/unionbedg/main.nf

```diff
@@ -0,0 +1,42 @@
+process BEDTOOLS_UNIONBEDG {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::bedtools=2.30.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bedtools:2.30.0--h7d7f7ad_2':
+        'quay.io/biocontainers/bedtools:2.30.0--h7d7f7ad_2' }"
+
+    input:
+    tuple val(meta), path(bedgraph, stageAs: "?/*")
+    path chrom_sizes
+    val(empty)
```

Comment:
`val(empty)` is for an optional value. These shouldn't be in the main inputs but should be given to the module through `ext.args`. 
P.S. you forgot the meta for `chrom_sizes` again ;)

Reply:
I didnt forget... it is an optional parameter, and when it was with meta I couldn't pass an empty value as []

Reply:
I removed the `chorm_sizes`, it's not mandatory, only if one specifies "-empty"

Reply:
You should include all optional files in the input too (otherwise you have no way to give them when needed). If the output is optional you can use `[[], []]` as input to that channel :)